assets/src/components/ladderPage.tsx

```diff
@@ -31,18 +29,12 @@ import {
 import OldCloseButton from "./oldCloseButton"
 import { SaveIcon, PlusThinIcon } from "../helpers/icon"
 import { tagManagerEvent } from "../helpers/googleTagManager"
-import useAlerts from "../hooks/useAlerts"
-import { SocketContext } from "../contexts/socketContext"
 import { fullStoryEvent } from "../helpers/fullStory"
 import { usePanelStateFromStateDispatchContext } from "../hooks/usePanelState"
 
 type DrawerContent = "route_picker" | "presets"
 
-export const findRouteById = (
-  routes: Route[] | null,
-  routeId: RouteId
-): Route | undefined => (routes || []).find((route) => route.id === routeId)
-
+// This is tested but never used?
```

Comment:
Wait is it even tested?

Reply:
Aye aye aye, I deleted the tests and didn't delete the function def, which is especially foolish! I meant to leave all references to the function intact and raise the question in the PR. Because it is unused, I'm leaning towards deletion and will do so unless instructed otherwise